portfolio/src/main/java/com/google/sps/servlets/CommentsController.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.util.Date;
```

Comment:
nit: import order, use ASCII sort https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Solved!

But, I'm a little bit confused because the following line in the styleguide:

> Within each block the imported names appear in ASCII sort order. (Note: this is not the same as the import statements being in ASCII sort order, since '.' sorts before ';'.)

I think I'm not catching the difference, it would be great if you can show me an example of a case when the order can vary